nomad/acl_endpoint.go

```diff
@@ -121,12 +121,36 @@ func (a *ACL) ListPolicies(args *structs.ACLPolicyListRequest, reply *structs.AC
 	defer metrics.MeasureSince([]string{"nomad", "acl", "list_policies"}, time.Now())
 
 	// Check management level permissions
-	if acl, err := a.srv.ResolveToken(args.AuthToken); err != nil {
+	acl, err := a.srv.ResolveToken(args.AuthToken)
+	if err != nil {
 		return err
-	} else if acl == nil || !acl.IsManagement() {
+	} else if acl == nil {
 		return structs.ErrPermissionDenied
```

Comment:
Doesn't `acl == nil` mean "ACLs not enabled"?

Not a big deal; an error is an error. Just seeing if there's a more precise one to use.

Reply:
There is a check above that checks if acl's are enabled and returns a better error. So this is actually an error that will never get hit.